alacritty/res/glsl3/text.v.glsl

```diff
@@ -39,12 +39,22 @@ void main() {
     // Position of cell from top-left
     vec2 cellPosition = cellDim * gridCoords;
 
-    if (backgroundPass != 0) {
+    fg = vec4(textColor.rgb / 255.0, textColor.a);
+
+    float occupiesCells = 1;
+    if ((int(fg.a) >= HAS_WIDE_CHAR)) {
+        // Update wide char x dimension so it'll cover the following spacer.
+        occupiesCells *= 2;
+
+        // Since we don't perform bitwise operations due to limitations of
+        // other renderers we subtract wide char bits keeping only colored.
+        fg.a = round(fg.a - HAS_WIDE_CHAR);
```

Comment:
Not sure what you mean by this, are bitwise operations not supported on gles2 with extension?

Reply:
>bitwise operations not supported on gles2 with extension

gles2 shaders don't have bitwise operations at all. It doesn't matter if you have extension or not.

Reply:
Could you state this explicitly rather than just mentioning "other renderers"? It's not exactly clear what that means here.